docker-alfresco/test/.env

```diff
@@ -1,7 +1,7 @@
 ALFRESCO_TAG=latest
 TRANSFORMERS_TAG=2.1.1
 SHARE_TAG=6.2.0
-SOLR6_TAG=1.4.0
+SOLR6_TAG=1.4.1
```

Comment:
This was automatically brought into the PR when creating. This change was added to master in #510

Reply:
We should focus this PR to only the changes required for the release stage, so this should be changed so that it does not change anything on the branch you are merging to (develop)